List by tag (#5)
FXinnovation/azure_metrics_exporter: README.md

Metrics of all matched resources are ignored (defaults to exclude none)
Excludes take precedence over the include filter.

### Resource tag filtering

> Resources having a specific tag can be filtered:

Reviewer: Maybe add more precision: 
```suggestion
Resources having a specific tag name and tag value can be filtered:
```

`resource_tag_name`:
Name of the tag to be filtered against.

`resource_tag_value`:

> Value that must me matched for the tag name passed.

Reviewer: Maybe more simply:
```suggestion
Value of the tag to be filtered against.
```
